src/js/components/TaskTable.js

```diff
@@ -196,7 +189,10 @@ class TaskTable extends React.Component {
 
 TaskTable.propTypes = {
   className: React.PropTypes.string,
-  parentRouter: React.PropTypes.func.isRequired,
+  parentRouter: React.PropTypes.oneOfType([
```

Comment:
There is a `JestUtil` to wrap the component so that you can use the context to get the router.